packages/instantsearch.js/src/widgets/hits/__tests__/hits.test.tsx

```diff
@@ -741,6 +741,41 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/hits/js/"
       `);
     });
 
+    test('provides __hitIndex in the list to each item', async () => {
```

Comment:
Why is this important to have this test?

Reply:
This ensures we keep track of an InstantSearch.js specific change in the hit data provided to the item template. It previously was tested in https://github.com/algolia/instantsearch/blob/5edf56bcadaccbc6d91f465ad710385ce869b3de/packages/instantsearch.js/src/components/Hits/__tests__/Hits-test.tsx#L186-L213 and I moved this to here as I removed obsolete test files.